### ISSUE TITLE
The package is not working with webpack build / aot

### ISSUE THREAD

**Issue by robertohuertasm** [NathanaelA/nativescript-localstorage#1]
I was trying to use your package but when I tried to bundle the app I received this error

```
ERROR in ../~/nativescript-localstorage/localstorage.js
Module not found: Error: Can't resolve 'file-system/file-system-access.js' in 'D:\GIT\gitlab\robertohuertasm\ti
mecontrol-tns\node_modules\nativescript-localstorage'
 @ ../~/nativescript-localstorage/localstorage.js 9:25-69
 @ ./app.module.ts
 @ ./app.module.ngfactory.ts
 @ ./main.aot.ts
```
I'm using nativescript cli 2.5.0 / android

**Comment by dzfweb** [comment]
The same error occurred with me.
#https://github.com/NativeScript/nativescript-dev-webpack/issues/81

**Comment by NathanaelA** [comment]
Remove the `.js` in the package.json file from the main key.     Sorry, this was something that NativeScript-Webpack added as a requirement recently.

**Comment by NathanaelA** [comment]
I have released v1.1.1 which fixes this issue.
